### ISSUE TITLE
Pandoc and OPS custom markdown extensions

### ISSUE THREAD

**Issue by annedranowski** [MicrosoftDocs/Contribute#246]
Hi,

I am using the Docs Authoring Pack for VS Code.

The extra functionality that is of greatest use to me is including files: `[!INCLUDE[sample include file](sample.md)]`.

The preview works great, rendering the included files. How should I build? to HTML, or pdf, or LaTeX.. Neither markdown-it nor pandoc recognize the include syntax.

Thanks!

Related questions are:

- Why are alerts not parsed/styled in preview? e.g.

```
> [!IMPORTANT]
> Essential information required for user success
```

shows up as `[!IMPORTANT] Essential information required for user success` indented.

- How do I use TOC generators with includes? the Markdown TOC extension will produce a table of contents that does not include headings in included files.

**Comment by nschonni** [comment]
>     * Why are alerts not parsed/styled in preview? e.g.

That was fixed in https://github.com/microsoft/vscode-docs-authoring/issues/200

**Comment by annedranowski** [comment]
> > ```
> > * Why are alerts not parsed/styled in preview? e.g.
> > ```
> 
> 
> That was fixed in [microsoft/vscode-docs-authoring#200](https://github.com/microsoft/vscode-docs-authoring/issues/200)

Thanks! I just needed to reload.

**Comment by JasonWHowell** [comment]
Hi @annedranowski 

There is the Docfx system which is open source to build pages from markdown. https://dotnet.github.io/docfx/

What is your business goal? These tools are meant for publishing on docs.microsoft.com. So it may not be designed to accommodate custom publishing workflows.

I'm not familiar if includes and the other extensions like the Important boxes are part of that platform natively. I think the docs extensions go further than Pandoc would know, since its something above and beyond standard Markdown. 

I don't think we have an open source build system for the current docs build platform. The Docs platform uses a different flavor of markdown now - Markdig.  

Thanks, Jason
